nadpisanie sekcji contacts

MichalJurek opened the pull request:
Poprzez nadpisanie 100% sekcji contacts oraz elementu contacts.scss wprowadzam szkielet sekcji. Wstępnie pokolorowanej i ułożonej. Kiedy zmiany zostaną zatwierdzone w kolejnym kroku, pokoloruje sekcje wedle kolorów ze zmiennych.
Zwracam uwagę na przesunięcie kart poprzez translateY . Dzięki temu uniosłem elementy zgodnie z makietą w FIGMA. Warunkuje to na sekcję, która znajduje się powyżej, gdyż zabiera trochę jej przestrzeni (dokładnie 166px) i analogicznie na sekcje poniżej zostawiając 166px do dyspozycji.

Przyciski, które znajdują się na dole sekcji contacts można by powiązać ze wspólną klasą buttons, gdyż większyść przycisków wygląda bardzo podobnie. Pozostawiam do decyzji lidera.

Tutaj można podejrzeć efekt w wersji live:
https://michaljurek.github.io/goit-markup-hw-08/portfolio.html

Pozdrawiam,
MJ

MichalJurek committed:
nadpisanie sekcji

Poprzez nadpisanie 100% sekcji contacts oraz elemntu contacts.scss wprowadzam szkielet sekcji. Wstępnie pokolorowanej i ułożonej. Kiedy zmiany zostaną zatwierdzone w kolejnym kroku, pokoloruje sekcje wedle kolorów ze zmiennych.

MateuszCharysz commented on src/sass/components/_contacts.scss:
```diff
@@ -4,12 +4,13 @@
     background-color: rgba(255, 245, 246, 1);
```
tutaj zamiast background-color: tło powinno być gradientem na 2 kolory

MichalJurek replied:
Faktycznie, poprawiam.

MateuszCharysz commented on src/sass/components/_contacts.scss:
```diff
@@ -4,12 +4,13 @@
     list-style: none;
     padding: 0;
-    margin: 0;
+    transform: translateY(-166px);
```
translate robi masakrę w tym momencie w sąsiednich sekcjach. Czy uzgodniłeś to z Mikołajem Marcinkowskim (footer) i Moniką Cichocką? (customer reviews)

MichalJurek replied:
Razem z gradientem przesunięcie translateY(-166px) nie jest potrzebne, wyciągam z repo.

MateuszCharysz reviewed (commented):
Czy uzgodniłeś zmiany w sąsiednich sekcjach z ich autorami?
Tekst zawija Ci się inaczej niż w figmie w wersji tabletowej (paddingi)

MichalJurek committed:
Update _contacts.scss

Poprawka tła.
Teraz jako gradient odpowiednio ostylowany przy zmianie między mobile a desktop.
Usunięcie funkcji translate, która wpływała na sekcje nad oraz pod. Teraz zamyka się we własnych granicach.

MateuszCharysz reviewed (approved):
Ok. Do ogarnięcia paddingi przy rozdzielczościach tabletowych

MichalJurek commented:
"paddingi przy rozdzielczościach tabletowych"

masz na myśli wewnątrz kart czy zewnętrze dla całego kontenera?

MateuszCharysz commented:
generalnie tego super nie sprawdzałem - po prostu tekst wewnątrz kart zawija się inaczej niż na obrazku w figmie - więc chyba wewnątrz kart

MichalJurek commented:
wrzuciłem kolejny pull, z omawianymi poprawkami
